test/asyncapi_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	testserver "github.com/confluentinc/cli/test/test-server"
 )
 
-func (s *CLITestSuite) TestAsyncApiExport() {
+func (s *CLITestSuite) TestAsyncApi_Export() {
```

Comment:
The existing way is preferred, since the command is named `confluent async-api export`. If we wanted to add more context to this test name, it might look like `TestAsyncApiExport_MoreContext()`

Reply:
The reasoning for this is you might want to run all of the `confluent asyncapi export` commands, so `make integration-test INTEGRATION_TEST_ARGS="-run TestCLI/TestAsyncapiExport" would run any of the tests with the prefix "TestAsyncapiExport".

Reply:
```suggestion
func (s *CLITestSuite) TestAsyncapiExport() {
```

Reply:
Oh, ok. I misunderstood the discussion. I'll do another pass and tidy this up.